src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values() [index].toString();

```

Comment:
Please, do not use toString() with enum
See [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)